solr/core/src/test/org/apache/solr/handler/TestContainerPlugin.java

```diff
@@ -548,16 +473,28 @@ public void m2(SolrQueryRequest req, SolrQueryResponse rsp) {
     }
   }
 
+  private Callable<V2Response> getPlugin(String path) {
```

Comment:
Is there a `.forceV2(true)` needed here, sometimes? `V2Request.Builder.forceV2` javadocs say "Only for testing. It's always true otherwise" but that doesn't seem to be true actually?

```suggestion
  private Callable<V2Response> getPlugin(String path, bool forceV2) {
```

Reply:
I think it's always needed to preserve the previous behavior, this was an oversight on my part.

Reply:
I added randomization to either forceV2 or not, and manually tested it both ways. I'm not sure what the intent was before, but let's improve test coverage while we're in the area.